lib/backburner/index.js

```diff
@@ -28,7 +28,7 @@ export default function Backburner(queueNames, options) {
 
   var _this = this;
   this._boundClearItems = function() {
-    clearItems();
+    clearItems.apply(_this, arguments);
```

Comment:
The whole reason for this regression was changing to avoid `bind`, right? If phantom is no longer an issue then that'd be okay I suppose.